sdk/servicebus/azure-servicebus/azure/servicebus/__init__.py

```diff
@@ -3,16 +3,18 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # -------------------------------------------------------------------------
+from uamqp import constants
 
 from ._version import VERSION
 __version__ = VERSION
 
-
-from azure.servicebus.common.message import Message, BatchMessage, PeekMessage, DeferredMessage
-from azure.servicebus.servicebus_client import ServiceBusClient, QueueClient, TopicClient, SubscriptionClient
-from azure.servicebus.common.constants import ReceiveSettleMode, NEXT_AVAILABLE
-from azure.servicebus.common.utils import AutoLockRenew
-from azure.servicebus.common.errors import (
+from ._servicebus_client import ServiceBusClient
+from ._servicebus_sender import ServiceBusSender
+from ._servicebus_receiver import ServiceBusReceiver
+from ._base_handler import ServiceBusSharedKeyCredential
+from .common.message import Message, BatchMessage, PeekMessage, ReceivedMessage
```

Comment:
should "common" be renamed to "_common"?

Reply:
updated `common` to `_common`